src/Files.OpenDialog/FilesLauncher/ServiceProviderBase.cpp

```diff
@@ -1,6 +1,5 @@
-// Copyright (C) Explorer++ Project
-// SPDX-License-Identifier: GPL-3.0-only
-// See LICENSE in the top level directory
+// Copyright (c) 2023 Files
+// Licensed under the MIT License. See the LICENSE.
```

Comment:
Is this work in progress? Even if you change the headers, it cannot be licensed under anything other than GPL as long as the original code remains. If you want to remove the GPL code, you must completely rewrite or delete the code.

Reply:
I know and I discussed everything about licensing with Files owner. I changed this to all files. and As you can see PR header comment, I have tasks and will remove those files.

Reply:
OK, I understand that they will eventually be removed.